code/game/gamemodes/clock_cult/clock_effects/clock_sigils.dm

```diff
@@ -117,6 +117,9 @@
 		return
 	if(is_eligible_servant(L))
 		to_chat(L, "<span class='heavy_brass'>\"You belong to me now.\"</span>")
+		if(!GLOB.script_scripture_unlocked)
+			GLOB.script_scripture_unlocked = TRUE
+			hierophant_message("<span class='large_brass bold'>A new Servant has been converted. Script scripture has been unlocked!</span>")
```

Comment:
why does this exist when the ticker will process and unlock script scripture if the requirements are set properly